packages/angular-cli/commands/init.run.ts

```diff
@@ -1,7 +1,10 @@
 import * as chalk from 'chalk';
+import * as check from '../utilities/check-package-manager';
```

Comment:
Normally unless there's a conflict you should `import {functionName} from '...'`.